common/testdata/testDataForCryptoutils/RSA_Cert.pem

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
@SVKREP Could you please cite the source of this file?

Reply:
I've used this command to generate the certificate:

```openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509  -out certificate.pem```

Reply:
@seshapad